src/shared/localShadowRepo.ts

```diff
@@ -278,16 +279,15 @@ export class ShadowRepo {
         } catch (e) {
           if (e instanceof git.Errors.MultipleGitError) {
             this.logger.error(`${e.errors.length} errors on git.add, showing the first 5:`, e.errors.slice(0, 5));
-            const error = new SfError(
-              e.message,
-              e.name,
-              [
+            throw SfError.create({
+              message: e.message,
+              name: e.name,
+              data: e.errors.map((err) => err.message),
```

Comment:
the previous code was saving the whole iso-git errors array but I think this is ok, those are still accessible in `e.cause.errors` in caller code.

Reply:
yeah, cause is meant to cover that.  `data` is typed now as AnyJson and TS didn't like `Error[]` (because error is a class, has various methods, etc).

Which is probably an improvement.